setup.cfg

```diff
@@ -52,6 +52,7 @@ test =
     ipython
     coverage
     skyfield
+    dask[array]
```

Comment:
dask should also be added to `all`, below

Reply:
@bsipocz - I don't think it needs to be, because it's not that astropy functionality works/doesn't work if the package is not available - this functionality would only be relevant if the user had dask installed and was creating arrays anyway. So that's why I put it here instead?

Reply:
How is it different then from pandas and h5py, while we list those as dependencies? So far, my understanding is at least, we list everything that gets imported somewhere in code.

Reply:
Yes good point - I'll add it

Reply:
Done!